src/rust/ide/view/src/debug_scenes/interface.rs

```diff
@@ -83,7 +83,7 @@ impl DummyTypeGenerator {
 fn init(app:&Application) {
 
     ensogl_theme::dark::setup(&app);
-    // ensogl_theme::light::setup(&app);
+    ensogl_theme::light::setup(&app);
```

Comment:
I understand now, that after commenting the light line, the `interface.rs` does not work, because dark classes in CSS would not register? If Im right, please add here the correct API usage for switching themes.

Reply:
Light one is the default theme and as far as i remember only default theme generates "vars" aside from theme data. Thus it wouldnt work properly, it needs to be the first one, and then the dark one should follow it.